modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/index.tsx

```diff
@@ -159,6 +161,15 @@ const CustomView: React.FC<React.HTMLAttributes<HTMLElement>> = () => {
 
 		const parentWindow = Liferay.Util.getOpener();
 
+		if (objectView.name[defaultLanguageId] === '') {
```

Comment:
Use `invalidateRequired()` in order to enforce same rules followed in the `BasicInfoScreen.tsx`
```suggestion
		if (invalidateRequired(objectView.name[defaultLanguageId])) {
```